zilencer/views.py

```diff
@@ -149,12 +153,15 @@ def register_remote_server(
         else:
             if not constant_time_compare(remote_server.api_key, zulip_org_key):
                 raise InvalidZulipServerKeyError(zulip_org_id)
-            else:
```

Comment:
This `else` was redundant and just makes the block super-indented. We have an early `raise` above.